bin/python/combine_reports.py

```diff
@@ -132,17 +134,23 @@ def _convert_iso(timestamp):
     GCP = None
     if CONFIG.get('gcp_cred') and CONFIG.get('from_gcp'):
         GCP = GcpManager(CONFIG, None)
-    assert all([attr in CONFIG for attr in ('from_time', 'to_time', 'device')]), """
+    else:
+        assert not CONFIG.get('from_gcp'), 'missing gcp_cred definition'
+
+    assert 'device' in CONFIG, """
 Combines reports under inst/reports(default) or from GCP
-Usage: combine_reports_from_date_range.py
+Usage: combine_reports.py
     [local/system.yaml]
     device=xx:xx:xx:xx:xx:xx
-    from_time='YYYY-MM-DDThh:mm:ss'
-    to_time='YYYY-MM-DDThh:mm:ss'
+    [from_time='YYYY-MM-DDThh:mm:ss']
+    [to_time='YYYY-MM-DDThh:mm:ss']
+    [daq_run_id=<nonce>]
     [count=N]
-    [from_gcp='true']
+    [from_gcp='true/false']
```

Comment:
any string passed here (including false) will be considered true under CONFIG.get('from_gcp'). We can either check if the string is true / false, or revert the doc to only indicate true as an option.

Reply:
Got it... actually had that fixed but sitting in an unpushed commit on my machine!